### PR TITLE
Allow for just funds to be passed in for market orders

### PR DESCRIPTION
**Pull request by poffdeluxe**
Fixes https://github.com/coinbase/gdax-node/issues/11

### CODE REVIEW

**Comment by awitherow** [comment]
Is there anything in the documentation (readme.md) about these changes you've made?

File: lib/clients/authenticated.js [hunk 1]
```diff
@@ -111,17 +111,24 @@ _.assign(AuthenticatedClient.prototype, new function() {
   prototype._placeOrder = function(params, callback) {
     var self = this;
 
-    var requiredParams = ['size', 'side', 'product_id'];
+    var requiredParams = ['side', 'product_id'];
 
     if (params.type !== 'market') {
       requiredParams.push('price');
+      requiredParams.push('size');
```
Comment (awitherow) [inline lib/clients/authenticated.js]: What happens when the params type is market and size is passed into the params anyways?

Comment (poffdeluxe) [inline lib/clients/authenticated.js]: In addition to a `funds` parameter? My understanding according to the API reference is that size will be used to place the order but it'll limit the buy according to funds.

>When placing a market order you can specify funds and/or size. Funds will limit how much of your quote currency account balance is used and size will limit the bitcoin amount transacted.

**Comment by poffdeluxe** [comment]
Yep! I'll try to expand documentation and tests a lil bit. Will put something together this week. May need to go back and forth with y'all to figure out the best way to format/expand the docs.

**Comment by poffdeluxe** [comment]
Also gonna squash my branch and fix my commit messages

File: lib/clients/authenticated.js (hunk 1 above)
Comment (awitherow) [inline lib/clients/authenticated.js]: 👍 I was just going through and doing some code reviews for notifications. I am not super knowledgable about the functionality there so I trust you 😃

**Comment by fb55** [comment]
Fixed in 93f19ac

**Comment by fb55** [comment]
Thanks a lot @poffdeluxe for the PR!

**Comment by fb55** [comment]
Also, feel free to open another PR if you get around to extend docs and/or tests

**Comment by poffdeluxe** [comment]
@fb55 Will do!

**Pull request closed without merging**